wiki/scripts/build.sh

```diff
@@ -34,15 +34,20 @@ rebuild() {
 	echo -e "$(date) $GREEN Updating docs for branch: $1.$RESET"
 	# Generate new docs after merging.
 
+	dir=''
```

Comment:
Just add a comment here that latest version is built in the public folder. Also, check that the dropdown still works when you try to change to latest version from some other version.